src/model/migrations.ts

```diff
@@ -638,6 +639,24 @@ export default async function runMigrations() {
 
   migrations.push(v18);
 
+  /*
+   *************** Migration v19 ******************
+    Move favorites from local storage to react query mmkv
+   */
+  const v19 = async () => {
+    const favoritesMetadata = await legacyStorage.get(['favorites']);
```

Comment:
i know its confusing cause we have 2 still but all new migrations should go under `@/migrations` this is the legacy shiz

that being said i dont think we need this since we are reading from async if theres not mmkv storage in the query persister 

would love to verify that is working without a migration and remove this if we can

Reply:
removed!